config.py

```
@@ -77,9 +243,27 @@ class DeviceMeta(TypedDict):
+    "12R": "oneplus_12r",
+    "11": "oneplus_11",
+    "11R": "oneplus_11r",
+    "10 Pro": "oneplus_10_pro",
+    "10T": "oneplus_10t",
+    "10R": "oneplus_10r_80w",
+    "9 Pro": "oneplus_9_pro",
+    "9": "oneplus_9",
+    "9RT": "oneplus_9rt",
+    "9R": "oneplus_9r",
+    "Pad 3": "oneplus_pad_3",
+    "Pad 3": "oneplus_pad_3",
```

Ruff (0.14.14)

[error] Dictionary key literal `"Pad 3"` repeated

Remove repeated key literal `"Pad 3"`

(F601)